src/LazyOperations/InverseLinearMap.jl

```diff
@@ -0,0 +1,333 @@
+
+julia> ilm2.M == B*A
+true
+```
+
+The application of a `InverseLinearMap` to a `ZeroSet` or an `EmptySet` is
+simplified automatically.
+
+```jldoctest constructors
+julia> InverseLinearMap(A, ZeroSet{Int}(3))
+ZeroSet{Int64}(3)
+```
+"""
+struct InverseLinearMap{N<:Real, S<:LazySet{N},
+                 NM, MAT<:AbstractMatrix{NM}} <: AbstractAffineMap{N, S}
+    M::MAT
+    X::S
+
+    # default constructor with dimension match check
+    function InverseLinearMap(M::MAT, X::S;
+                                check_invertibility::Bool=true) where {N<:Real,
+                                    S<:LazySet{N}, NM, MAT<:AbstractMatrix{NM}}
+        @assert dim(X) == size(M, 1) "a linear map of size $(size(M)) cannot " *
+            "be applied to a set of dimension $(dim(X))"
+        if check_invertibility
+            @assert isinvertible(M) "the linear map is not invertible"
+        end
+        return new{N, S, NM, MAT}(M, X)
+    end
+end
+
+
+# convenience constructor from a UniformScaling
+function InverseLinearMap(M::UniformScaling{N}, X::LazySet) where {N}
+    if M.λ == one(N)
+        return X
+    end
+    return InverseLinearMap(Diagonal(fill(M.λ, dim(X))), X)
+end
+
+
+# convenience constructor from a scalar
+function InverseLinearMap(α::Real, X::LazySet)
+    n = dim(X)
+    return InverseLinearMap(sparse(α * I, n, n), X)
+end
+
+# combine two linear maps into a single linear map
+function InverseLinearMap(M::AbstractMatrix, ilm::InverseLinearMap)
+    return InverseLinearMap(ilm.M * M, ilm.X)
+end
+
+# ZeroSet is "almost absorbing" for InverseLinearMap (only the dimension changes)
+function InverseLinearMap(M::AbstractMatrix{N}, Z::ZeroSet{N}) where {N}
+    @assert dim(Z) == size(M, 2) "a linear map of size $(size(M)) cannot " *
+            "be applied to a set of dimension $(dim(Z))"
+    return ZeroSet{N}(size(M, 1))
+end
+
+# EmptySet is absorbing for LinearMap
+function InverseLinearMap(M::AbstractMatrix, ∅::EmptySet)
+    return ∅
+end
+
+
+# --- AbstractAffineMap interface functions ---
+
+function matrix(ilm::InverseLinearMap)
+    return ilm.M
+end
+
+function vector(ilm::InverseLinearMap{N}) where {N}
+    return spzeros(N, dim(ilm))
+end
+
+function set(ilm::InverseLinearMap)
+    return ilm.X
+end
+
+"""
+    dim(ilm::InverseLinearMap)
+
+Return the dimension of an inverse linear map.
+
+### Input
+
+- `ilm` -- inverse linear map
+
+### Output
+
+The ambient dimension of the inverse linear map.
+"""
+function dim(ilm::InverseLinearMap)
+    return size(ilm.M, 1)
+end
+
+"""
+    σ(d::AbstractVector{N}, ilm::InverseLinearMap{N}) where {N<:Real}
+
+Return the support vector of the inverse linear map.
+
+### Input
+
+- `d`   -- direction
+- `ilm` -- inverse linear map
+
+### Output
+
+The support vector in the given direction.
+If the direction has norm zero, the result depends on the wrapped set.
+
+### Notes
+
+If ``L = M^{-1}⋅S``, where ``M`` is a matrix and ``S`` is a convex set, since
```

Comment:
can you use ``L = M^{-1}⋅X``? so the notation in the docstring and the code are the same.

Reply:
@dfcaporale  here and in other places, we usually mark conversations as resolved only *after* they have been fixed. otherwise, it is easy to forget what still needs attention :)

Reply:
if you want to comment that you agree (but will do later), the thumbsup :+1:  should be fine!

Reply:
You are right. What happened is that I modified the file localy with your comments and didn't want to loose them. I'll be more careful in the following.